todo-app/src/stores/TaskStore.js

```diff
@@ -0,0 +1,52 @@
+import {autorun, observable, action} from 'mobx';
+import {loadTodoItems, saveTodoItems} from '../storage'
+
+
+export default class TaskStore {
+  constructor() {
+    autorun(() => {
+      saveTodoItems(this.todos);
+    });
+  }
+    
+    @observable
+    todos = loadTodoItems();
+    
+
+    @action
+    addTask(text){
```

Comment:
You refer to your "things" as both "todos" and "tasks". It would be less confusing if you chose one term and stuck with it.